tableau-databricks/oauth-config.xml

```diff
@@ -11,6 +11,7 @@
 
     <authUri>/oauth2/v2.0/authorize</authUri>
     <tokenUri>/oauth2/v2.0/token</tokenUri>
+    <instanceUrlValidationRegex>^https:\/\/([a-zA-Z0-9-.]+\.)?((microsoftonline\.(com|us|cn|de))|chinacloudapi\.cn)\/(.*)</instanceUrlValidationRegex>
```

Comment:
This is for additional security enhancement to guarantee we will not go to a malicious url even someone open the twb and manually change the instanceUrl and send to a victim.